Task №2

jacsmedvedev opened the pull request:
First commit of this task.

jacsmedvedev committed:
Task №2

First commit of this task.

strazhkoe commented on src/task_from_teacher_2/Student.java:
```diff
@@ -0,0 +1,124 @@
+	 * @param studentMarks
+	 * @return scholarship with premium
+	 */
+	public double NewScholarships(int[] studentMarks) {
```
Как посчитает стипендию для студента с оценками 5, 4, 3, 3, 3?

jacsmedvedev committed:
Task №2

Second commit of this task.
Changes:
1.improved logic of calculation of the scholarships;
2.method toString() overridden in class student
